docs/notebooks/DataObfuscation.ipynb

```diff
@@ -1716,4 +1716,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 4
-}
```

Comment:
Seems odd that these changes (there's another later on) made their way in. Not 100% sure whether newline at the end of the file is needed.